js/main.js

```diff
@@ -31,17 +31,28 @@ window.addEventListener('submit', function (event) {
     xhr.open('GET', url);
     xhr.responseType = 'json';
     xhr.addEventListener('load', function () {
+      if (xhr.status !== 200) {
+        window.alert('No data has been received');
```

Comment:
Don't use alerts, they can be disabled by the browser for some users, you want either a modal or some other way to display the message to the user.